test/e2e/upgrades/ingress.go

```diff
@@ -93,10 +97,11 @@ func (t *IngressUpgradeTest) Setup(f *framework.Framework) {
 		"kubernetes.io/ingress.global-static-ip-name": t.ipName,
 		"kubernetes.io/ingress.allow-http":            "false",
 	}, map[string]string{})
-	jig.AddHTTPS("tls-secret", "ingress.test.com")
+	framework.DescribeIng(jig.Ingress.Namespace)
```

Comment:
Do we still need this describe? Or if we are keeping it, move it to after `AddHTTPS()`?

Reply:
Removed.